test/search_test.dart

```diff
@@ -220,15 +220,46 @@ void main() {
       });
 
       test('facetDistributions parameter', () async {
+        //setup
         var index = await createBooksIndex();
         var response = await index
             .updateSettings(IndexSettings(
-              filterableAttributes: ['tag'],
+              filterableAttributes: [ktag],
             ))
             .waitFor(client: client);
-        expect(response.status, 'succeeded');
+        //action
```

Comment:
This comment is unnecessary if you split the test phases with blank lines! My suggestion is to remove them :)